### ISSUE TITLE
Between filter behaviour

### ISSUE THREAD

**Issue by Naghal** [osTicket/osTicket#5155]
### Prerequisites

* [ No] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [No error ] Do you have any errors in the PHP error log, or javascript console?
* [x ] Did you check the [osTicket forums](https://forum.osticket.com/)?
* [x ] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

### Description

The filtering using the "between" method is not as expected. 

### Steps to Reproduce

1. Do an advanced search
2. Filter any date field by "between" and select for example 2019-04-11 - 2019-05-11

**Expected behavior:** Data between the interval is filtered, as well as 2019-04-11 and 2019-05-11

**Actual behavior:**  Data between the interval is filtered, as well as 2019-04-11, but **NOT 2019-05-11**

### Versions

OsTicket | v1.12.3 (bcf1a6f) 
Logiciel serveur Web | Apache/2.2.15 (CentOS)
Version de MySQL | 5.5.52
Version PHP | 7.3.11

Running on Windows 10 on latest Chrome

**Comment by TechCare-Aus** [comment]
I checked this on my system too, running the following:

> osTicket Version | v1.14-rc2-104-g3f42833c (3f42833)  
> Web Server Software | Apache
> MySQL Version | 10.3.19
> PHP Version | 7.3.11
> Timezone | AEDT (Interpreted as Australia/Melbourne)

Same issue.

Ticket came in on 04/11/19 7:14 am AWST. When I do an advanced search on create date, set to:

- 4/11/2019 to 4/11/2019. It finds nothing.
- 3/11/2019 to 4/11/2019. It finds nothing.
- 3/11/2019 to 5/11/2019. It finds the ticket.
- 4/11/2019 to 5/11/2019. It finds the ticket.

My server is hosted in GMT +10, I've set osTicket to GMT +8

Also side note, when selecting a date the option is to click (in my case) AWST. But this doesn't do anything. Should it? It is a hyperlink.
